vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/format/NoticeEventFormatter.kt

```diff
@@ -103,28 +105,54 @@ class NoticeEventFormatter @Inject constructor(private val sessionHolder: Active
     private fun formatRoomCreateEvent(event: Event): CharSequence? {
         return event.getClearContent().toModel<RoomCreateContent>()
                 ?.takeIf { it.creator.isNullOrBlank().not() }
-                ?.let { sp.getString(R.string.notice_room_created, it.creator) }
+                ?.let {
+                    if (event.isSentByCurrentUser()) {
+                        sp.getString(R.string.notice_room_created_by_you)
+                    } else {
+                        sp.getString(R.string.notice_room_created, it.creator)
+                    }
+                }
     }
 
     private fun formatRoomNameEvent(event: Event, senderName: String?): CharSequence? {
         val content = event.getClearContent().toModel<RoomNameContent>() ?: return null
         return if (content.name.isNullOrBlank()) {
-            sp.getString(R.string.notice_room_name_removed, senderName)
+            if (event.isSentByCurrentUser()) {
```

Comment:
I didn't like these if-else statements, but don't have a better approach at the moment.

Reply:
Yes, it's not ideal...